### PR TITLE
Add Scaleway stack variant

### PR DESCRIPTION
**Pull request by sdurrheimer**
Tested on:

*   [x] Scaleway (stack)
*   [x] AWS (stack)
*   [x] Scaleway (extra)
*   [x] AWS (extra)

### CODE REVIEW

File: ansible/packer.yml [hunk 1]
```diff
@@ -23,29 +23,40 @@
...
-        dest: /home/admin/user-data.sh
-        owner: admin
-        group: admin
+        dest: "{{ install_user_home }}/first-boot.yml"
+        owner: "{{ install_user }}"
+        group: "{{ install_user_group }}"
+    
+    - block:
+      - template:
+          src: user-data.sh.scw.tpl
```
Comment (gaelL) [inline ansible/packer.yml:49]: Is there a way to keep only one template if they are quite similar ?

File: pipeline/pipeline-scw.yml [hunk 2]
```diff
@@ -0,0 +1,386 @@
+# YAML anchors
```
Comment (gaelL) [inline pipeline/pipeline-scw.yml:1]: Do we need an entier new pipeline or could we rely on our latest way to give all terraform config as variable to keep only one pipeline ?

Comment (sdurrheimer) [reply pipeline/pipeline-scw.yml:1]: we can't, terraform backend needs extra parameters to work with scaleway

File: ansible/packer.yml (hunk 1 above)
Comment (sdurrheimer) [reply ansible/packer.yml:49]: Too complicated to know that we are in a Scaleway use case inside the script.
It is way simpler to know it on Ansible side.

**Review by gaelL — APPROVED** [review]

**Pull request merged**